system/db/features.txt

```diff
@@ -80,6 +80,9 @@
 		{ name: "ClericHealPartySelect", enabled: true },
 	]},
 	
+	// Enables storages to have multiple stacks of same item type
+	{ name: "StorageMultiStack", enabled: true },
```

Comment:
The features file is intended for things that changed over the course of the game's life, like official changes to skills or NPCs, how properties worked, or things were calculated. The idea is that it should be easy to, for example, enable or disable all rebuild changes, or picking out a couple you like. Custom features on the other hand should go into the conf files.